subsys/usb/class/netusb/Kconfig

```diff
@@ -12,6 +12,7 @@ menu "USB Device Networking support"
 config USB_DEVICE_NETWORK
 	bool
 	default n
+	select NET_L2_ETHERNET
```

Comment:
Do both ECM and RNDIS works on Ethernet level? (ECM I know it does, but RNDIS?)

Reply:
I would not use select here (reverse dependency) but depends.

"select should be used with care. select will force a symbol to a value without visiting the dependencies. By abusing select you are able to select a symbol FOO even if FOO depends on BAR that is not set. In general use select only for non-visible symbols (no prompts anywhere) and for symbols with no dependencies."

Reply:
But then we need to include NET_L2_ETHERNET to every sample, this is also following pattern with SLIP:
```
config  SLIP_TAP
	bool "Use TAP interface to host"
	default y
	select NET_L2_ETHERNET

```

Reply:
Yes, select seems fair after all (no hidden dependency) and simpler from user perspective.